onebusaway-android/src/main/java/org/onebusaway/android/ui/NavigationDrawerFragment.java

```diff
@@ -82,28 +80,15 @@
     protected static final int NAVDRAWER_ITEM_SEND_FEEDBACK = 5;
 
     protected static final int NAVDRAWER_ITEM_PLAN_TRIP = 6;
-    
-    // 7 is reserved for re-adding the Embedded Social "Popular" item - See #889
-    //protected static final int NAVDRAWER_ITEM_POPULAR = 7;
 
-    protected static final int NAVDRAWER_ITEM_PINS = 8;
+    protected static final int NAVDRAWER_ITEM_OPEN_SOURCE = 7;
 
-    protected static final int NAVDRAWER_ITEM_ACTIVITY_FEED = 9;
```

Comment:
So what happens if someone has one of the ES screens as their most recently used screen, installs the update, and then starts the app? Seems like currently some strange behavior would happen. 

I suggest we leave all these ES `int` values here, mark them as deprecated, and then handle them in this class and `HomeActivity` by loading up the `NAVDRAWER_ITEM_NEARBY` when ES screen was selected. That way someone who previously has an ES screen as most recently used would then end up in the "Nearby" view.

Reply:
Good catch; these values need to be reserved.